easy-ui-react/src/Button/Button.tsx

```diff
@@ -58,8 +59,18 @@ export function Button(props: ButtonProps) {
     { ...props, elementType: As },
     ref,
   );
+
+  const bothIconPropsDefined = iconAtEnd && iconAtStart;
+  if (bothIconPropsDefined) {
+    // eslint-disable-next-line no-console
+    console.warn("Cannot simultaneously define `iconAtEnd` and `iconAtStart`");
+  }
+
   const canUseIcon =
-    (iconAtEnd || iconAtStart) && variant !== "link" && size !== "sm";
+    (iconAtEnd || iconAtStart) &&
+    !bothIconPropsDefined &&
```

Comment:
**nitpick:** since `bothIconPropsDefined` triggers an error above, you could remove this from the check, but not a huge deal either way